packages/contracts-periphery/tasks/install-drippie-config.ts

```diff
@@ -0,0 +1,75 @@
+import { task } from 'hardhat/config'
+import { LedgerSigner } from '@ethersproject/hardware-wallets'
+import { PopulatedTransaction } from 'ethers'
+
+import { DripConfig, getDrippieConfig } from '../src'
+
+task('install-drippie-config').setAction(async (args, hre) => {
+  console.log(`connecting to ledger...`)
+  const signer = new LedgerSigner(
+    hre.ethers.provider,
+    'default',
+    hre.ethers.utils.defaultPath
+  )
+
+  console.log(`connecting to Drippie...`)
+  const Drippie = await hre.ethers.getContractAt(
+    'Drippie',
+    (
+      await hre.deployments.get('Drippie')
+    ).address,
+    signer
+  )
+
+  console.log(`loading local version of Drippie config for network...`)
+  const config = await getDrippieConfig(hre)
+
+  // Need this to deal with annoying Ethers/Ledger 1559 issue.
+  const sendtx = async (tx: PopulatedTransaction): Promise<void> => {
+    const gas = await signer.estimateGas(tx)
```

Comment:
This looks like it does gas estimation twice since populate tx is called right before. Maybe instead do gas estimation if its not defined?